priv/repo/migrations/20200424141938_create_apikeys.exs

```diff
@@ -5,9 +5,10 @@ defmodule Auth.Repo.Migrations.CreateApikeys do
     create table(:apikeys) do
       add :client_id, :binary
       add :client_secret, :binary
-      add :name, :string
-      add :description, :text
-      add :url, :binary
+      # add :name, :string
+      # add :description, :text
+      # add :url, :binary
+      add :app_id, references(:apps, on_delete: :nothing)
```

Comment:
Should Api Keys get deleted if the parent app gets deleted?

Reply:
We want to "deactivate" any keys. And We won't be deleting any data for now. See: #106